circuit-types/src/keychain.rs

```diff
@@ -26,8 +26,8 @@ use super::{biguint_from_hex_string, biguint_to_hex_string};
 pub const NUM_KEYS: usize = 4;
 /// The number of bytes used in a single scalar to represent a key
 pub const SCALAR_MAX_BYTES: usize = 31;
-/// The number of words needed to represent a non-native root key
-pub const ROOT_KEY_WORDS: usize = 2;
+/// The number of words needed to represent a non-native root key coordinate
```

Comment:
Can we expand this doc comment to indicate that we are representing a curve point under the hood? "coordinate" alone may not make sense without context